src/Microsoft.Health.Dicom.Blob/Features/Telemetry/BlobMeter.cs

```diff
@@ -0,0 +1,28 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.Metrics;
+using Microsoft.Health.Dicom.Core.Features.Telemetry;
+
+namespace Microsoft.Health.Dicom.Blob.Features.Telemetry;
+public sealed class BlobMeter : IDisposable
```

Comment:
Let's be consistent and have two meters for this: `BlobStoreMeter` and `BlobRetrieveMeter`. We should maintain the pattern of operation-based meters. Similarly, their names should be like `$"{OpenTelemetryLabels.BaseMeterName}.Store.Blob"`

Reply:
Renamed to `BlobStoreMeter`